crypto/src/mls/conversation/export.rs

```diff
@@ -0,0 +1,149 @@
+// Wire
+// Copyright (C) 2022 Wire Swiss GmbH
+
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+
+//! Primitives to export data from a group, such as derived keys and client ids.
+
+use mls_crypto_provider::MlsCryptoProvider;
+
+use crate::mls::{client::ClientId, ConversationId, CryptoError, CryptoResult, MlsCentral, MlsConversation, MlsError};
+
+const EXPORTER_LABEL: &str = "exporter";
+
+impl MlsConversation {
+    /// See [MlsCentral::export_secret_key]
+    pub fn export_secret_key(&self, backend: &MlsCryptoProvider, key_length: usize) -> CryptoResult<Vec<u8>> {
+        self.group
+            .export_secret(backend, EXPORTER_LABEL, &[], key_length)
```

Comment:
Variabilize `&[]` into a `EXPORTER_CONTEXT` const and add a TODO for reminding ourselves to make sure this can be a constant